### ISSUE TITLE
doesn't build

### ISSUE THREAD

**Issue by fartwhif** [Mag-nus/Mag-Plugins#4]
The type or namespace name 'Util' could not be found (are you missing a using directive or an assembly reference?)

**Comment by immortalbob** [comment]
looks like it needs decal.adapter, and a bunch of virindi stuff to build...

**Comment by nbucciarelli** [comment]
I'm having the same issue. decal.adapter + the virindi stuff is all included. Any luck @fartwhif ?

**Comment by fartwhif** [comment]
To get it to compile I used pinvoke.net to pull in the missing interop stuff, renamed some utility stuff, some updates to user32 class, util class, and SettingsFile class, and
commented out the saving/loading of default character.  Sorry, I don't have a copy.

Since then I talked to Thwargle and he told me about his copy, which might fit your needs: https://github.com/Thwargle/ThwargLauncher/tree/master/ThwargLauncher/Mag-Software/Mag-Filter

Thwargle did say I got him curious as to why this isn't building, for what it's worth.
